Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -116,7 +116,7 @@ private void Destroy(Object obj)
 #if UNITY_EDITOR
         Object.DestroyImmediate(obj);
 #else
-        Object.Destroy(actions);
+        Object.DestroyImmediate(actions);
```

Comment:
According to documentation of Object.DestroyImmediate and Object.Destroy that should not be done? https://docs.unity3d.com/ScriptReference/Object.DestroyImmediate.html